test/providers/provider_level_zero.cpp

```diff
@@ -2,11 +2,12 @@
 // Under the Apache License v2.0 with LLVM Exceptions. See LICENSE.TXT.
 // SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 
-#include <dlfcn.h>
-#include <level_zero/ze_api.h>
+#define NOMINMAX
```

Comment:
why it is needed?

Reply:
It is needed to disable the default min and max macros from Windows

Reply:
as it's only for windows move it under ifdef - and/or extend it with information that it's for windows

Reply:
Okey